src/plays/hangman-game/styles.css

```diff
@@ -0,0 +1 @@
+/* enter stlyes here */
```

Comment:
keep all styled-components here & export them to use in respective files

Reply:
@sudipkundu999  done!